core/src/client/async_client/mod.rs

```diff
@@ -551,11 +551,9 @@ async fn handle_backend_messages<S: TransportSenderT, R: TransportReceiverT>(
 			handle_recv_message(raw.as_ref(), manager, sender, max_notifs_per_subscription).await?;
 		}
 		Some(Err(e)) => {
-			tracing::error!("Error: {:?} terminating client", e);
 			return Err(Error::Transport(e.into()));
 		}
 		None => {
-			tracing::error!("[backend]: WebSocket receiver dropped; terminate client");
 			return Err(Error::Custom("WebSocket receiver dropped".into()));
```

Comment:
can you fix this as well?

It is wrong as it has been moved to the `async client`

Reply:
This was solved by committing suggestions! Thanks!